### PR TITLE
Fix punctuation in `os.execvpe` docstring.

### PR DESCRIPTION
**Pull request by hramezani**
Fix punctuation in `os.execvpe` docstring.

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

**Review by mangrisano — APPROVED** [review]
Thank you.

File: Lib/os.py [hunk 1]
```diff
@@ -568,7 +568,7 @@ def execvpe(file, args, env):
     """execvpe(file, args, env)
 
     Execute the executable file (which is searched for along $PATH)
-    with argument list args and environment env , replacing the
+    with argument list args and environment env, replacing the
     current process.
     args may be a list or tuple of strings. """
```
Comment (hansrajdas) [inline Lib/os.py:573]: nit: Please remove extra space from end of doc string also.

Comment (hramezani) [reply Lib/os.py:573]: We have this extra space in all function docstring. for example:
https://github.com/python/cpython/blob/24fe46081be3d1c01b3d21cb39bc3492ab4485a3/Lib/os.py#L532
https://github.com/python/cpython/blob/24fe46081be3d1c01b3d21cb39bc3492ab4485a3/Lib/os.py#L555

Comment (hansrajdas) [reply Lib/os.py:573]: Strange, don't know this is intentional or what. In that case, we should take this trailing space removal in a separate PR. Thank you!

**Review by hansrajdas — APPROVED** [review]

**Pull request merged**